eval/src/vespa/eval/tensor/dense/dense_tensor_address_combiner.h

```diff
@@ -79,6 +79,8 @@ class DenseTensorAddressCombiner
 
     template <typename Func>
     void for_each(const AddressContext & rightAddress, const CellsRef & rhsCells, Func && func) {
+        // The rightAddress oly holds the starting point for iteration and what is need to efficiently maintain
```

Comment:
I suggest moving this comment to the AddressContext class itself, since it also holds for the combined address...